src/pages/studyView/StudyViewPage.tsx

```diff
@@ -75,62 +76,57 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
-                                   styleOpts={{
-                                       width: 450,
-                                       height: 300
-                                   }}
-                                   fileName="Overall_Survival"/>
+                        unalteredPatientSurvivals={data.unalteredGroup}
+                        title={'test'}
+                        xAxisLabel="Months Survival"
+                        yAxisLabel="Surviving"
+                        totalCasesHeader="Number of Cases, Total"
+                        statusCasesHeader="Number of Cases, Deceased"
+                        medianMonthsHeader="Median Months Survival"
+                        yLabelTooltip="Survival estimate"
+                        xLabelWithEventTooltip="Time of death"
+                        xLabelWithoutEventTooltip="Time of last observation"
+                        showDownloadButtons={false}
+                        showTable={false}
+                        showLegend={false}
+                        styleOpts={{
+                            width: 450,
+                            height: 300
+                        }}
+                        fileName="Overall_Survival" />
                 </StudyViewComponentLoader>
             </div>
         </div>
     }
 
     render() {
-        let cancerStudy = this.store.studyMetaData.result!;
-        return (
-            <div className="studyView">
-                <div className="topBanner">
-                    {
-                        this.store.studyMetaData.isComplete && (
-                            <div className="studyViewHeader">
-                                <h3>{cancerStudy.name}</h3>
-                                <p dangerouslySetInnerHTML={{__html: cancerStudy.description}}></p>
-                            </div>
-                        )
-                    }
-                </div>
-                <MSKTabs id="studyViewTabs" activeTabId={this.props.routing.location.query.tab}
-                         className="mainTabs">
-
-                    <MSKTab key={0} id="summaryTab" linkText="Summary">
-                            {
-                                this.store.defaultVisibleAttributes.isComplete && 
-                                this.store.initalClinicalDataCounts.isComplete &&
-                                this.store.initialized && 
-                                (
-                                    <div className={styles.studyViewFlexContainer}>
-                                        {this.store.visibleAttributes.map(this.renderAttributeChart)}
-                                    </div>
-                                )
-                            }
-                            {
-                                <div className={styles.studyViewFlexContainer}>
-                                    {this.store.survivalPlotData.result.map(this.renderSurvivalPlot)}
+        if (this.store.studies.isComplete) {
```

Comment:
Should we add conditions when isPending and isError?

Reply:
I can add this in up coming pr, by showing a loading symbol while its still pending